miden-lib/src/transaction/procedures/kernel_v0.rs

```diff
@@ -1,74 +1,74 @@
 /// This file is generated by build.rs, do not modify
 
-use miden_objects::{digest, Digest, Felt};
+use miden_objects::{digest_from_hex, Digest, Felt};
 
 // KERNEL V0 PROCEDURES
 // ================================================================================================
 
 /// Hashes of all dynamically executed procedures from the kernel 0.
 pub const KERNEL0_PROCEDURES: [Digest; 32] = [
     // account_vault_add_asset
-    digest!(0xa9fe41a58d609bc6, 0xca49c09c352213a4, 0x30f55868815fc5, 0x84f2f2540b3d335e),
+    digest_from_hex!("0xc69b608da541fea9a41322359cc049cac55f816858f530005e333d0b54f2f284"),
```

Comment:
Should there be a test for these macros and its validations? Maybe comparing the old digest macro with the new one as well

Reply:
I thought about it, but in fact every test which internally uses the kernel procedures is checking the correctness of this macro, otherwise it will fail. That's why I wasn't sure whether do we need to create a dedicated tests for this macro.